xml/admin_saltcluster.xml

```diff
@@ -351,6 +351,37 @@ role-rgw/cluster/rgw1*.sls</screen>
 &prompt.smaster;salt-run rescinded.ids
 &prompt.smaster;salt-run state.orch ceph.stage.5</screen>
   </example>
+  <example xml:id="ex-failed-node">
+   <title>Removal of a Failed Node</title>
+   <para>
+     If the &sminion; is not responding and the administrator is unable to
+     resolve the issue, we recommend removing the &salt; key and leaving the
+     server intact and still operating in the capacity as a storage node.
```

Comment:
I think the "and leaving... node" can be removed.  The succinct "we recommend removing the salt key" gets to the point without creating other questions for the admin (e.g. what if it's not a storage node?).